libs/partners/snowflake/docs/retrievers.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
Is it possible to move this file to `docs/docs/integrations/retrievers/` folder where all retriever examples live?